src/pages/settings/InitialPage.js

```diff
@@ -103,6 +122,23 @@ const InitialSettingsPage = ({
     if (_.isEmpty(myPersonalDetails)) {
         return null;
     }
+
+    // Add free policies (workspaces) to the list of menu items
+    const workspaceMenuItems = _.chain(policies)
+        .filter(policy => (
+            !_.isEmpty(policy)
```

Comment:
NAB, why would there be empty policies (assuming that means `{}`)?

Reply:
And in that case probably it would not have type or role and would get filtered?

Reply:
Yeah, that was just included as a null check but I think it's unnecessary and can be removed.